TourMate/TourMate/Frontend/Trip/Views/TripView/TripView.swift

```diff
@@ -38,15 +38,7 @@ struct TripView: View {
                             .padding([.bottom, .horizontal])
 
                         if let imageUrl = viewModel.trip.imageUrl {
```

Comment:
`imageUrl` would always exist given our current implementation?

Reply:
yep, but its an optional field now.. maybe we can change it in next sprint